packages/babel/src/index.js

```diff
@@ -45,6 +45,7 @@ const unpackInputPluginOptions = ({ skipPreflightCheck = false, ...rest }) => {
       supportsStaticESM: true,
       supportsDynamicImport: true,
       supportsTopLevelAwait: true,
+      supportsExportNamespaceFrom: true,
```

Comment:
I think it should be possible to make this conditional (from Rollup's perspective) to avoid changing peer dep range and thus avoid this being a breaking change. I bet it should be rather easily possible to read Babel's version somehow as well, right?

Reply:
We don't need detect Babel version, undeclared caller flags will be considered as noop for preset-env.

I have updated PR to check rollup versions, however it seems that I can not retrieve version from `rollup/package.json`.